src/core/inline-idl-parser.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Parses an IDL string and returns its components as:
+ *
```

Comment:
Did you end up adding internal slot support: 

```
Foo[[bar]]
```

Reply:
Ah, see it below. Should we document it here?

Reply:
`Foo[[bar]]` is also treated as internal slot? I have just added support for `[[bar]]` till now.

Reply:
So yeah, `Foo[[bar]]` should be:

```
<a>Foo</a><a>[[\bar]]</a>
```

Internal slots are always for specific interfaces.